tds/src/main/webapp/WEB-INF/classes/resources/xsl/ncssSobs.xsl

```diff
@@ -161,7 +161,7 @@
             </div>
         </form>
         <hr/>
-        <h3><a href="http://www.unidata.ucar.edu/software/netcdf-java/stream/CdmRemote.html">CDM Remote Documentation</a></h3>
+        <h3><a href="httpss://www.unidata.ucar.edu/software/netcdf-java/stream/CdmRemote.html">CDM Remote Documentation</a></h3>

```

Comment:
Looks like there is an extra `s` in this one. Also, the link should be https://www.unidata.ucar.edu/software/netcdf-java/v4.6/reference/stream/NcStream.html (that one that is there 404s).

Reply:
D'oh.  Sorry.  Fixed in commit https://github.com/Unidata/thredds/commit/6b07552d90aeedc063fbb3d4a8115dabfdefa768